tests/php/RestApi/Utilities/ProductSummary.php

```diff
@@ -31,6 +31,22 @@ public function test_get_summary() {
 		$this->assertEquals( "<p>Lorem ipsum dolor&hellip;</p>\n", $class->get_summary( 3 ) );
 		// Should return 1 word.
 		$this->assertEquals( "<p>Lorem&hellip;</p>\n", $class->get_summary( 1 ) );
+
+		// Test for languages where characters are words.
+		add_filter(
+			'gettext_with_context',
+			function( $translated, $text, $context ) {
+				if ( $text === 'words' && $context === 'Word count type. Do not translate!' ) {
+					return 'characters';
+				}
+				return $translated;
+			},
+			10,
+			3
+		);
```

Comment:
I can't remember, are all implemented filters cleared on `tearDown` in WP tests or do they need to be explicitly removed?

Reply:
Good catch, solved in https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/1639/commits/923b938481d990cce3733951f01b4f87a8f3c22e